msbuild/Xamarin.MacDev.Tasks.Core/SiriIntentsKeys.cs

```diff
@@ -0,0 +1,69 @@
+﻿//
+// SiriIntentsKeys.cs
+//
+// Author: Jeffrey Stedfast <jestedfa@microsoft.com>
+//
+// Copyright (c) 2018 Microsoft Corp.
+//
+
+namespace Xamarin.MacDev.Tasks
+{
+	static class SiriIntentsKeys
+	{
+		public const string INIntentDefinitionModelVersion = "INIntentDefinitionModelVersion";
```

Comment:
Minor - if the name of the C# symbol and the value of the string are the same, you can do this:
`public const string INIntentDefinitionModelVersion = nameof (INIntentDefinitionModelVersion);`